packages/tools/devtools/devtools-browser-extension/src/messaging/fence.json

```diff
@@ -1,5 +1,5 @@
 {
 	"tags": ["messaging"],
 	"exports": ["index"],
-	"imports": []
+	"imports": ["utilities"]
```

Comment:
Not strictly required for this PR, but I discovered this was missing while testing some changes I ultimately abandoned.